keps/sig-scheduling/2926-job-mutable-scheduling-directives/README.md

```diff
@@ -162,8 +162,8 @@ specific partition (e.g., preemptibles) or a specific location (e.g., zone x).
 
 This is a proposal to relax update validation on jobs that have never been unsuspended to 
 allow mutating node scheduling directives, namely the pod template's node affinity, 
-node selector, tolerations, annotations and labels of the job's pod template. This enables 
-a higher-level queue controller to inject such directives before un-suspending a job to
+node selector, tolerations, schedulingGates, annotations and labels of the job's pod template. This 
```

Comment:
@Huang-Wei @soltysh  including the new schedulingGates as well.

Reply:
Friendly ping.

Reply:
So it will need validation changes to guarantee this for schedulingGates, right?

Reply:
yes, just like the others